index.js

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env node
```

Comment:
I reintroduced this because it was useful for debugging some differences when running proxy.js vs. index.js